test-vault/tasks/Tasks Completed on specific Date.md

```diff
@@ -0,0 +1,5 @@
+## Completed on a specific date
+```dataview
+task from "tasks" where completedDate and
+completedDate=date(2021-08-06)
```

Comment:
This can just be `TASK FROM "tasks" WHERE completedDate = date(2021-08-06)`; the null check is implicit in the equality.